### PR TITLE
Campaigns fix

### PR DESCRIPTION
**Pull request by soleti**
This PR includes:
- Local version of the job types for quick testing (not used by default)
- CORSIKA mixing campaign
- Bug fixes (path for mcs files)

### CODE REVIEW

File: JobConfig/mixing/CreatePBISequence.fcl [hunk 1]
```diff
@@ -12,6 +12,7 @@ services: {
 
 source: {
   module_type :  PBISequence
+  reconstitutedModuleLabel : PBISim
```
Comment (brownd1978) [inline JobConfig/mixing/CreatePBISequence.fcl:15]: I don't know why this shows up as a difference, this line is already merged.

**Review by brownd1978 — COMMENTED** [review]
I believe this PR depends on other pending PRs in Production and Offline, so it shouldn't be merged yet.

File: JobConfig/mixing/CreatePBISequence.fcl (hunk 1 above)
Comment (soleti) [reply JobConfig/mixing/CreatePBISequence.fcl:15]: Yeah, I also noticed this, I am not sure

**Comment by brownd1978** [comment]
I guess it doesn't formally depend on the other PRs, but to produce the output we want it needs changes to the OffSpill trigger and removal of the SignalFilter from mixing.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**